src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensRangeHandler.cs

```diff
@@ -50,8 +50,8 @@ public SemanticTokensRangeHandler()
             // document request, so caching range results is unnecessary since the whole document
             // handler will cache the results anyway.
             var (tokensData, isFinalized) = await SemanticTokensHelpers.ComputeSemanticTokensDataAsync(
-                context.Document, SemanticTokensHelpers.TokenTypeToIndex,
-                request.Range, cancellationToken).ConfigureAwait(false);
+                context.Document, SemanticTokensHelpers.TokenTypeToIndex, request.Range,
```

Comment:
Nit: Personally I'm of the opinion that parameters should be all on the same line or each on their own line to avoid diffs like this.